app/models/chargeback_rate.rb

```diff
@@ -11,6 +20,8 @@ class ChargebackRate < ApplicationRecord
   validates_uniqueness_of   :guid
   validates_uniqueness_of   :description, :scope => :rate_type
 
+  virtual_column :assigned?, :type => :boolean
```

Comment:
Here's a more complete example:

```ruby
class User < ActiveRecord::Base
  before_destroy :nope

  def nope
    errors.add :id, "no destroying this!"
    throw :abort
  end
end

x = User.create
p x.id
user = User.find(x.id)
if user.destroy
else
  # couldn't destroy the user object
  p user.errors
end
```

In the example, `destroy` should return false because the callback failed.  If the caller checks the return value of `destroy`, then they can check `errors` on the model (as usual).